source/sass/buyers-guide/views/category.scss

```diff
@@ -39,6 +39,24 @@
         padding-right: 3%;
       }
 
+      &.draft {
```

Comment:
This is duplicated in a few places mostly because there's no design yet and the final CSS will likely be different from what this does.